ui/tests/test_ui.py

```diff
@@ -0,0 +1,169 @@
+import os
```

Comment:
When we require having a backend running, we could move this file to the python repo (where the dependencies are already declared) and have it run from there.

The tests could be annotated with a special pytest marker for e2e or ui-testing.